src/transformers/utils/import_utils.py

```diff
@@ -749,6 +749,7 @@ def is_torch_compile_available():
 def is_torchdynamo_compiling():
     if not is_torch_available():
         return False
+    return True
```

Comment:
this is a quick fix, as this function is missing support for CUDA graph tracing.